packages/tooling/openmrs/src/cli.ts

```diff
@@ -230,6 +233,12 @@ yargs.command(
       .describe(
         "importmap",
         "The import map to use. Can be a path to an import map to be taken literally, an URL, or a fixed JSON object."
+      )
+      .boolean("download-coreapps")
+      .default("download-coreapps", false)
+      .describe(
+        "download-coreapps",
+        "Allows automatically downloading the core app shell apps if they are not currently avaialble."
```

Comment:
```suggestion
        "Downloads and bundles the core apps. For cases where the core apps are not in the import map."
```

Do I understand this correctly?

Reply:
Yeah, that reads much better.